src/osemgrep/language_server/scan_helpers/Diagnostics.ml

```diff
@@ -37,7 +37,7 @@ let diagnostic_of_match is_intellij (m : OutJ.cli_match) =
   let diagnostic =
     Diagnostic.create
       ~range:(Conv.range_of_cli_match m)
-      ~code ~severity ~source:"Semgrep" ~message
+      ~code ~severity ~source:"Semgrep" ~message:(`String message)
```

Comment:
No need to change this here, but @ajbt200128 we probably want this to be markdown, right?

Reply:
I thought that'd be nice too when I saw the feature but I didn't want to introduce anything that could be considered breaking in this PR.
Happy to do so if there's something in mind or leave it for a different PR

Reply:
Feel free to leave it; I fear what to set here is potentially more complex than always using one or the other for consistency with how we render elsewhere.

Reply:
this won't be a breaking change in anyway, so let's just leave this as markdown. appreciate the care to detail though!

Reply:
hi @ajbt200128 so you prefer to change from String to MarkdownContent?

I chose `String` of the 2 new options because I believe before it wasn't markdown, at least in some tests I did, there were no bolds

Just let me know which one and I'll update

Reply:
As-is is fine.